pkgs/development/tools/analysis/rizin/default.nix

```diff
@@ -60,6 +60,7 @@ let rizin = stdenv.mkDerivation rec {
     ./librz-wrapper-support.patch
 
     ./0001-fix-compilation-with-clang.patch
+    ./0002-disable-pcre2-jit.patch
```

Comment:
Can you link this to an issue, so we know when we can potentially drop this patch?

Reply:
If this is an update that needs to be done in Cutter, than a good place would be in their upstream repository.

Reply:
I think the relevant issue is https://github.com/NixOS/nixpkgs/issues/300056.
Because for anything else then NixOS it works fine.

If switching to PCRE 10.44 is the fix, it will be done when the new Cutter releases. But this might take a while.

The patch here is more of a hotfix. Because it just disables JIT completely.

Can you give me any hint about the build issue? We could also instruct our users to build Cutter for themselves for the time being. But this would require simple build instructions IMHO.

Reply:
See suggestion below.